webapp/channels/src/components/threading/global_threads/thread_list/thread_list.tsx

```diff
@@ -196,12 +213,19 @@ const ThreadList = ({
             <Header
                 id={'tutorial-threads-mobile-header'}
                 heading={(
-                    <>
+                    <div
+                        className='tab-buttons-list'
+                        aria-label={formatMessage({
+                            id: 'threading.threadList.tabsLabel',
+                            defaultMessage: 'Filter visible threads',
+                        })}
+                        {...tabListProps}
+                    >
                         <div className={'tab-button-wrapper'}>
                             <Button
                                 className={'Button___large Margined'}
                                 isActive={currentFilter === ThreadFilter.none}
-                                onClick={handleRead}
+                                {...tabProps[0]}
```

Comment:
I wish we could use a clearer identifier rather than a numerical index, but I suppose it would probably complicate things in the new hook.

Reply:
I'm not super happy with the the numberical index here, but I think it's okay. Ideally, I think we'd be using `tabProps` like `tabProps.map((tab) => <Button .../>)`, but because of the extra `CRTUnreadTutorialTip` that's on the second tab, I wasn't able to cleanly do that here